resources/js/Pages/Admin/BookingRequests/BookingRequestsList.vue

```diff
@@ -10,29 +10,37 @@
         <!-- Rooms List -->
         <template #content>
           <div class="space-y-6">
-            <div class="grid grid-cols-7">
+            <div class="grid grid-cols-8">
               <div class="text-md mx-3">User</div>
               <div class="text-md mx-3">Room</div>
               <div class="text-md mx-3">Start Time</div>
               <div class="text-md mx-3">End time</div>
               <div class="text-md mx-3">Status</div>
+              <div class="text-md mx-3">Reference</div>
             </div>
 
             <div
               v-for="booking_request in booking_requests"
               :key="booking_request.id"
               class="grid flex items-center"
             >
-              <div class="grid grid-cols-7">
+              <div class="grid grid-cols-8">
```

Comment:
I think this is about how wide we want the table to be since the new column is technically data and not an action button. If any more columns need to be added we should compress our action buttons into a dropdown like in RoomList.vue